mail/opensmtpd/Portfile

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           legacysupport 1.0
+
+github.setup        opensmtpd opensmtpd 6.8.0p2
+categories          mail
+platforms           darwin
+license             ISC
+maintainers         {@ryanakca debian.org:rak}
+description         secure, reliable, lean, and easy-to configure SMTP server
+long_description    The OpenSMTPD server seeks to be \
+                    \n* as secure as possible, and uses privilege separation to mitigate \
+                    \n  possible security bugs \
+                    \n* as reliable as possible: any accepted email must not be lost \
+                    \n* lean: it covers typical usage cases instead of every obscure one \
+                    \n* easy to configure, with a configuration syntax reminiscent of the OpenBSD \
+                    \n  Packet Filter's (PF) \
+                    \n* fast and efficient: it can handle large queues with reasonable performance
+homepage            https://www.opensmtpd.org
+
+github.tarball_from archive
+master_sites        https://opensmtpd.org/archives/
```

Comment:
`github.tarball_from` changes the default of `github.master_sites`, which is the default value of `master_sites`. It seems unnecessary to use both `github.tarball_from` and `master_sites`.

Reply:
The portfile used to get its tarballs from github, but I switched it a while back to use the official tarballs from the project's website. I've dropped the github portgroup and manually set `name` and `version` instead.